cmd/scollector/collectors/processes_windows.go

```diff
@@ -40,7 +40,7 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 	}
 
 	var svc_dst []Win32_Service
-	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	var svc_q = wmi.CreateQuery(&svc_dst, "")
```

Comment:
@alienth Oh also I use https://wmie.codeplex.com/ to explore these. Then we have some linqpad C# which generates the Go structs from C#. Somewhere in the repo